src/AFQMC/Hamiltonians/rotateHamiltonian.hpp

```diff
@@ -476,8 +477,7 @@ inline void rotateHijkl(std::string& type,
       assert(nkcum + K0_ == M_split[nn + nn0]);
       if (M_split[nn + nn0 + 1] == M_split[nn + nn0])
         continue;
-      int nblk       = Qknum[nn];
-      long ntermscum = 0;
```

Comment:
This was used in an assertion. Thus breaks Debug builds. Unfortunately, current CI Debug doesn't cover AFQMC.

Reply:
Good catch, would using `#ifndef NDEBUG` work? I see it's used in other parts of the code.

Reply:
`#ifndef NDEBUG` is likely a solution to keep the check for debug build without printing a warning in the release builds.

Reply:
Works for me also. Great to see the warning reductions.